packages/beacon-node/src/util/workerEvents.ts

```diff
@@ -34,6 +38,12 @@ export function wireEventsOnWorkerThread<EventData>(
       // This check is not necessary but added for safety in case of improper implemented events
       isWorkerToMain[data.event] === EventDirection.mainToWorker
     ) {
+      const [sec, nanoSec] = process.hrtime(data.posted);
```

Comment:
Node docs suggest that `hrtime` is deprecated (https://nodejs.org/api/process.html#processhrtimetime) and `hrtime.bigint()` should be used instead.